packages/onboarding/src/notice/style.scss

```diff
@@ -0,0 +1,19 @@
+.onboarding-notice {
+	display: flex;
+	color: var( --studio-gray-50 );
+	font-size: 0.875em; /* stylelint-disable-line */
+	margin-bottom: 1.5em;
+
+	svg {
+		align-self: center;
+		fill: var( --color-warning-30 );
+		flex-shrink: 0;
+		margin-right: 1em;
+		transform: rotate( 180deg );
```

Comment:
It's ok to rotate the `info` icon. Maybe a "warning outline" icon should be added to Dashicons. But this is outside the scope of this PR.